connector/mqtt/vernemq/k2v-bridge/README.md

```diff
@@ -13,76 +13,101 @@ If there are more instances than partitions, the exceeding ones will not consume
+not set the default values for all of them, but you can change the value of the ones we did not
+mentioned here via environment variables.
+
+Check the library's documentation for more information on the parameters.
+
+__NOTE THAT__ the module adapts the dotted version of the variables to the camelCase one that the
+library accepts, i.e. `client.id` is converted to `clientId`.
+
+Key | Default Value | Valid Values | Environment variable
+--- | ------------- | ------------ | --------------------
+mqtt.ca | /opt/k2v_bridge/app/cert/ca.crt | string | K2V_MQTT_CA
+mqtt.cert | /opt/k2v_bridge/app/cert/${K2V_APP_HOSTNAME:-k2v-bridge}.crt | string | K2V_MQTT_CERT
+mqtt.clean | false | boolean | K2V_MQTT_CLEAN
+mqtt.client.id | ${K2V_APP_HOSTNAME:-k2v-bridge} | string | K2V_MQTT_CLIENT_ID
+mqtt.host | vernemq-k8s | string | K2V_MQTT_HOST
+mqtt.keep.alive | 60 | integer | K2V_MQTT_KEEP_ALIVE
+mqtt.key | /opt/k2v_bridge/app/cert/${K2V_APP_HOSTNAME:-k2v-bridge}.key | string | K2V_MQTT_KEY
+mqtt.port | 8883 | integer | K2V_MQTT_PORT
+mqtt.protocol | mqtts | string | K2V_MQTT_PROTOCOL
+mqtt.reject.unauthorized | true | boolean | K2V_MQTT_REJECT_UNAUTHORIZED
+mqtt.username | ${K2V_APP_HOSTNAME:-k2v-bridge} | string | K2V_MQTT_USERNAME
+
+## **SDK Consumer**
+
+These parameters are passed directly to the SDK producer. Check the
+[official repository](https://github.com/dojot/dojot-microservice-sdk-js) for more info on the
+values.
+
+### **Main object**
+
+Key | Default Value | Valid Values | Environment variable
+--- | ------------- | ------------ | --------------------
+sdk.in.processing.max.messages | 1 | integer | K2V_SDK_IN_PROCESSING_MAX_MESSAGES
+sdk.queued.max.messages.bytes | 10485760 | integer | K2V_SDK_QUEUED_MAX_MESSAGES_BYTES
+sdk.subscription.backoff.min.ms | 1000 | integer | K2V_SDK_SUBSCRIPTION_BACKOFF_MIN_MS
+sdk.subscription.backoff.max.ms | 60000 | integer | K2V_SDK_SUBSCRIPTION_BACKOFF_MAX_MS
+sdk.subscription.backoff.delta.ms | 1000 | integer | K2V_SDK_SUBSCRIPTION_BACKOFF_DELTA_MS
+sdk.commit.interval.ms | 5000 | integer | K2V_SDK_COMMIT_INTERVAL_MS
+
+### **kafka.consumer object**
+
+Key | Default Value | Valid Values | Environment variable
+--- | ------------- | ------------ | --------------------
+consumer.client.id | ${K2V_APP_HOSTNAME:-k2v-bridge} | string | K2V_CONSUMER_CLIENT_ID
+consumer.group.id | k2v-bridge-group-id | string | K2V_CONSUMER_GROUP_ID
+consumer.max.in.flight.requests.per.connection | 1000000 | integer | K2V_CONSUMER_MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION
+consumer.metadata.broker.list | kafka-server:9092 | string | K2V_CONSUMER_METADATA_BROKER_LIST
+consumer.socket.keepalive.enable | false | boolean | K2V_CONSUMER_SOCKET_KEEPALIVE_ENABLE
+
+### **kafka.topic object**
+
+| Key | Default Value | Valid Values | Environment variable
+| --- | ------------- | ------------ | --------------------
+| topic.acks | -1 | integer | K2V_TOPIC_ACKS
```

Comment:
I think we could add here `'auto.offset.reset': 'beginning'` to force consumption from the beginning; otherwise, some messages might not be processed when a consumer groups is started for the first time.